packages/tldraw/src/lib/ui/context/actions.tsx

```diff
@@ -1111,7 +1127,6 @@ export function ActionsProvider({ overrides, children }: ActionsProviderProps) {
 					trackEvent('toggle-snap-mode', { source })
 					editor.user.updateUserPreferences({ isSnapMode: !editor.user.getIsSnapMode() })
 				},
-				checkbox: true,
```

Comment:
sorry, maybe i missed this in my read-through - this is captured somewhere else now?

Reply:
Nope - this was set here and then never used